### PR TITLE
Fix infinite-loading infinitely loading

### PR DESCRIPTION
**Pull request by Tirokk**
> [roschaefer](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-23T18:09:01Z" title="Friday, August 23rd 2019, 8:09:01 pm +02:00">Aug 23, 2019</time>_
_Merged <time datetime="2019-08-26T19:44:20Z" title="Monday, August 26th 2019, 9:44:20 pm +02:00">Aug 26, 2019</time>_
---

## 🍰 Pullrequest
The infinite loading component was indeed loading infinitely.

### Issues

- None

### Todo

- [X] None

### CODE REVIEW

File: webapp/pages/profile/_id/_slug.vue [hunk 1]
```diff
@@ -244,8 +244,10 @@
         <div
           v-if="hasMore"
           v-infinite-scroll="showMoreContributions"
-          infinite-scroll-disabled="$apollo.loading"
-          infinite-scroll-distance="10"
+          :infinite-scroll-disabled="$apollo.loading"
```
Comment (Tirokk) [inline webapp/pages/profile/_id/_slug.vue:247]: > [mattwr18](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-08-26T09:37:34Z" title="Monday, August 26th 2019, 11:37:34 am +02:00">Aug 26, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -238,8 +238,10 @@
         <div
           v-if="hasMore"
           v-infinite-scroll="showMoreContributions"
-          infinite-scroll-disabled="$apollo.loading"
-          infinite-scroll-distance="10"
+          :infinite-scroll-immediate-check="true"
```


what is the purpose of adding this?
doesn't this default to true anyways?
https://sroze.github.io/ngInfiniteScroll/documentation.html
or am I looking at a different library??

File: webapp/pages/profile/_id/_slug.vue [hunk 2]
```diff
@@ -244,8 +244,10 @@
...
+          :infinite-scroll-distance="10"
+          :infinite-scroll-throttle-delay="800"
```
Comment (Tirokk) [inline webapp/pages/profile/_id/_slug.vue:249]: > [mattwr18](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-08-26T09:39:56Z" title="Monday, August 26th 2019, 11:39:56 am +02:00">Aug 26, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -238,8 +238,10 @@
         <div
           v-if="hasMore"
           v-infinite-scroll="showMoreContributions"
-          infinite-scroll-disabled="$apollo.loading"
-          infinite-scroll-distance="10"
+          :infinite-scroll-immediate-check="true"
+          :infinite-scroll-disabled="$apollo.loading"
+          :infinite-scroll-distance="10"
+          :infinite-scroll-throttle-delay="800"
```


default is 200ms... what went into the thinking to make it `800`? just your feeling? or was there something specific?

Comment (Tirokk) [reply webapp/pages/profile/_id/_slug.vue:249]: > [roschaefer](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-26T10:10:08Z" title="Monday, August 26th 2019, 12:10:08 pm +02:00">Aug 26, 2019</time>_

---


It was try/error with @alina-beck. We tested it with seed data. Maybe we can increase it even further after we learned about the low performance of our neo4j pod.

File: webapp/pages/profile/_id/_slug.vue (hunk 1 above)
Comment (Tirokk) [reply webapp/pages/profile/_id/_slug.vue:247]: > [roschaefer](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-26T10:11:42Z" title="Monday, August 26th 2019, 12:11:42 pm +02:00">Aug 26, 2019</time>_

---


Right! I double-checked and it seems it is the default per documentation. I wonder why this fixed some our cypress tests earlier. Maybe another change or the documentation is just wrong.

Comment (Tirokk) [reply webapp/pages/profile/_id/_slug.vue:247]: > [roschaefer](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-26T11:57:05Z" title="Monday, August 26th 2019, 1:57:05 pm +02:00">Aug 26, 2019</time>_

---


Done, ping @mattwr18

**Review by Mogge — APPROVED** [review]
ok, so maybe the docs are wrong @roschaefer?? weird.. anyways, I don't care if we have this one line if it makes it work for us

**Pull request closed without merging**